.travis.yml

```diff
@@ -45,12 +45,10 @@ before_install:
     - export PATH=/usr/lib/jvm/java-8-oracle/jre/bin/:$PATH
     - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
     - java -version
-    - ant -version
```

Comment:
Can you replace this with ```mvn --version``` to make sure which version of maven is installed and which version of Java it is using ?